wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,58 @@
+
+You can appeal by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh). Again, we will need your username, and make sure that you are sending it from the email address attached to your account! After you send your appeal, expect a reply within one week. If you have not received one then, please contact us on twitter or by email - there is a chance we have not received it or closed it by accident!
+
+When trying to appeal, you are free to write what you want in your message! We do however have expectations for what the appeal should include. Appeals do not have to be full-length essays - as long as you are honest and genuine, your appeal will be sufficient. We are looking to make sure you know what you were restricted for and you are aware of what rules you have broken. There is no point in unrestricting a user who will continue breaking the rules after being unrestricted.
+
+We expect the following points in appeals:
+- **Account history.** It does not need to be too detailed. This should include all your breaches of the major rules (multi-accounting, account sharing, cheating, or major silences). You do not need to remember every date, every silence, or every account you created, as long as you are honest with what you did. This includes any rules you have broken while restricted if you did.
+- **Why you broke the rules.** An explanation as to what led you to decide to create new accounts, give your account away, cheat, or others, will help us (and you) determine whether or not you are at risk of offending again. If you continued offending while restricted, we will also be interested in knowing why.
+- **How you obtained the means to do so.** This applies mostly to cheating - we want to know where you obtained the tool you used, how you came across it, and whether or not it is still present on your computer.
+- **Why we should allow you back.** Most of the time, this ties in to the second point. We just want to ensure that you will not be breaking the rules again! Please understand that the heavier your account's history, the more we will expect from this section.
+
+## My appeal has been denied!
+
+There are only a few reasons why an appeal gets denied.
+- **It was sent too early.** If the cooldown for your account has not yet passed, your appeal will be denied. If you are unsure when you can actually appeal, please reply to our answer - we will give you the correct date.
+- **Your appeal was insufficient or incomplete.** If your appeal does not cover the four points listed above, we will request that you write a more proper one. This **does not** reset the cooldown for your account and you are free to send your new appeal at any point in time! If you attempted lying or hiding parts of your history (mainly multi-accounting or cheating), your appeal will be denied for being non-truthful and your cooldown reset to three months. This is another chance for you to reflect on your actions and make sure you did not forget anything.
```

Comment:
`- **Your appeal was insufficient or incomplete.** If your appeal does not sufficiently cover the four points listed above, we will request that you write a more complete one. This **does not** necessarily reset the cooldown for your account and you are free to send your new appeal at any point in time! However. if you attempted lying or hiding major rule violations you committed (mainly multi-accounting or cheating), your appeal will be denied for being non-truthful, and your cooldown will be reset to three months. This is another chance for you to reflect on your actions and make sure you did not forget anything.`